python/README.rst

```diff
@@ -48,104 +54,92 @@ to core deltachat library::
 
     git clone https://github.com/deltachat/deltachat-core-rust
     cd deltachat-core-rust
-    cargo build -p deltachat_ffi --release
+    cd python
 
-This will result in a ``libdeltachat.so`` and ``libdeltachat.a`` files
-in the ``target/release`` directory. These files are needed for
-creating the python bindings for deltachat::
+It is always a good idea to create a python "virtualenv".
+Install "virtualenv" on your system and run::
 
-    cd python
-    DCC_RS_DEV=`pwd`/.. pip install -e .
+    virtualenv venv
+    source venv/bin/activate
```

Comment:
Might make sense to put the `python3 -m venv` instructions here rather than referring to even more projects to install?  I know it's a little different but should be sufficient for most people and most will have python3 by now.

Reply:
not sure -- on my ubuntu system python -m venv does not just work but you have to install python3.5-ensurepip or something.